sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,117 @@
+}
+
+SHTC3Sensor::Builder& SHTC3Sensor::Builder::setLowPower(bool low_power) {
+    low_power_ = low_power;
+    return *this;
+}
+
+SHTC3Sensor SHTC3Sensor::Builder::build() const {
+    return SHTC3Sensor(address_, low_power_, scl_pin_, sda_pin_);
+}
+
+bool SHTC3Sensor::initializeBus() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)